src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -1163,7 +1160,6 @@ class SparqlQleverVisitor : public SparqlAutomaticVisitor {
       AD_CHECK(ad_utility::getLowercase(children[0]->getText()) == "sample");
       return makePtr.operator()<sparqlExpression::SampleExpression>();
     }
-    AD_FAIL()  // Should be unreachable.
```

Comment:
For consistency (this should also be in the coding style) as above:
reinstate the `AD_FAIL()` and make the last case an `else if` instead of the `AD_CHECK()`.